src/Polar/Constraints/line_flow.jl

```diff
@@ -30,13 +30,12 @@ function flow_constraints_grad!(polar::PolarForm, cons_grad, buffer, weights)
     fill!(cons_grad, 0)
     adj_vmag = @view cons_grad[1:nbus]
     adj_vang = @view cons_grad[nbus+1:2*nbus]
-    ev = adj_branch_flow_kernel!(polar.device)(weights, buffer.vmag, adj_vmag,
+    adj_branch_flow!(weights, buffer.vmag, adj_vmag,
```

Comment:
How about renaming this function `adjoint_branch_flow!`, for clarity?

Reply:
We agreed that there will be a future naming PR.